dev/MRTCore/mrt/mrm/mrmex/HNamesBuilder.cpp

```diff
@@ -0,0 +1,1600 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "StdAfx.h"
+#include "mrm/build/HNamesBuilder.h"
+#include "mrm/Checksums.h"
+
+namespace Microsoft::Resources::Build
+{
+
+// HNamesNode - common to ItemInfo and ScopeInfo
+
+HNamesNode::HNamesNode(__in const HierarchicalNamesConfig* pConfig) :
+    m_pParentScope(nullptr), m_segment(pConfig), m_index(-1), m_nameIndex(-1)
```

Comment:
initialize in declaration, (more instances below)
(non-blocking)

Reply:
legacy code

---
In reply to: [488887748](https://github.com/microsoft/ProjectReunion/pull/185#discussion_r488887748) [](ancestors = 488887748)